chrome/extension/create_styleurl/CreateStyleURL.js

```diff
@@ -276,15 +285,13 @@ class CreateStyleURLContainer extends React.Component {
     });
   };
 
-  handleHide = () => {
-    this.setState({
-      hidden: true
-    });
+  toggleBar = isVisible => {
+    this.setState({ hidden: !isVisible });
```

Comment:
IMO, this should be `setVisible` and then you pass it true/false to show/hide rather than to toggle, less likely to cause issues this way

Reply:
i dont understand what u mean here? like rename function setvisible?

Reply:
1. Rename the function to `setVisible`

2. Instead of:

Reply:
this.setVisible(true) would imply that it's visible no?  (hidden would be true)